dv/uvm/core_ibex/riscv_dv_extension/testlist.yaml

```diff
@@ -514,8 +514,6 @@
   iterations: 5
   no_iss: 1
   rtl_test: core_ibex_csr_test
-  sim_opts: >
-    +disable_cosim=1
```

Comment:
Leave it disabled on this test for now, it does self check after all.

We do want to enable it eventually for the CSR test but I suspect there's still a couple of CSR related mismatches and I'd prefer to have this test working (using it's self checking)).